slides/src/content/display.md

```diff
@@ -0,0 +1,148 @@
+<!-- .slide: data-background="../content/images/title-slide.jpg" -->
+
+## Building Applications with Angular
+
+# Displaying Data
+
+---
+
+## Make the Data
+
+- Change the title
+- Add a list of strings
+
+_src/app/app.component.ts_
+```ts
+@Component({
+  …as before…
+}
+export class AppComponent {
+  title = 'To Do';
+  thingsToDo = [
+    'Learn JavaScript',
+    'Learn Angular',
+    'Learn Redux'
+  ];
+}
+```
+
+---
+
+## Display the Data
+
+- Use `*ngFor` to repeat some HTML multiple times
+- The loop variable `item` is created locally
+
+_src/app/app.component.html_
+```html
+<h1>{{title}}</h1>
+<p *ngFor="let item of thingsToDo">{{item}}</p>
+```
+
+![ngFor Output](content/images/screenshot-ngfor.png)
+
```

Comment:
The image displayed does not correspond with the code on the slide (or previous slides), or with the code in Greg's fork of the todo app. Specifically, the items have no item number in parens.